### ISSUE TITLE
Cannot place caret outside of inline code

### ISSUE THREAD

**Issue by chrisdeeming** [editor-js/inline-code#38]
Write the following text:

> This is `inline code`.

Backspace to delete the `.` at the end of the line and the caret jumps into the inline code tag. You now cannot add normal text at the end of this line. Most editors which work in a similar way allow you to click outside of the inline code or use the arrow keys to place the caret outside.

**Comment by samandar-boymurodov** [comment]
Hello @chrisdeeming , did you manage to fix this problem?

**Comment by chrisdeeming** [comment]
No. I moved on to Tiptap instead. This editor didn't meet my needs.

**Comment by samandar-boymurodov** [comment]
Okay, I see
